addons/web/static/src/legacy/js/views/calendar/calendar_renderer.js

```diff
@@ -905,7 +905,7 @@ return AbstractRenderer.extend({
             const start = moment(event.extendedProps.r_start);
             const end = moment(event.extendedProps.r_end);
             const duration = moment.duration(end.diff(start)).asDays();
-            event.extendedProps.record.start_hour = event.extendedProps.record.allday === false && duration < 1 ? start.format("HH:mm") : "";
+            event.extendedProps.record.start_hour = !event.extendedProps.record.allday && duration < 1 ? start.format("HH:mm") : "";
```

Comment:
I am not sure the real problem is that we need to change this... 

The problem we have is that the model project.task does not have an allday field. But would it make sense and how do we compute it? If really needed and if no rationale can be found on the model, the best might be an override of the data fetch process in order to set that field to false (in our module).

Reply:
@IT-Ideas , In our case we have allday as `undefined` as we does not have such field. so instead of `== false` strict check, can't we check directly using `!allday`(true in case of undefined)? Here creating a new field does not make any sense. and to fetch the data it will directly call the search_read so do not prefer to override it. 

Thanks

Reply:
@mba-odoo no problem for the JS framework 🙂 
👍🏼